tests/integration/components/render-popover-new-api-test.js

```diff
@@ -18,6 +18,15 @@ function openModal(context, componentSpec, options={}) {
   return Ember.run(() => context.container.lookup('service:popover').openModal(componentName, options));
 }
 
+function openPopoverModal(context, componentName, options={}) {
+  return Ember.run(() => {
```

Comment:
Might we want to take this opportunity to update to `import { run } from '@ember/runloop';` here? I see `Ember.run` is used elsewhere in this file, but maybe we should get away from the Ember global in all 3 cases?

Reply:
@kpfefferle I don't believe this codebase is using the babel transform for the Ember module API

Reply:
Oh dang, good call 😬